### PR TITLE
Adds plates to the dinnerware vending machine

### PR DESCRIPTION
**Pull request by TurboGay**
As requested by Prize Bot, this PR adds 20 plates to the dinnerware vending machine so that chefs may better present their creations. Tested locally without any issues, hopefully there's no problems with this PR as it's my first one.

:cl:
* rscadd: Adds plates to the dinnerware vending machine.

### CODE REVIEW

**Review by ShiftyRail — APPROVED** [review]

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -2029,10 +2029,11 @@ var/global/num_vending_terminals = 1
 		/obj/item/weapon/reagent_containers/food/drinks/drinkingglass = 8,
 		/obj/item/clothing/suit/chef/classic = 2,
 		/obj/item/trash/bowl = 20,
+		/obj/item/trash/plate = 20,
 		/obj/item/weapon/reagent_containers/food/condiment/peppermill = 5,
 		/obj/item/weapon/reagent_containers/food/condiment/saltshaker	= 5,
 		/obj/item/weapon/reagent_containers/food/condiment/vinegar = 5,
-		/obj/item/weapon/storage/bag/food = 5
+		/obj/item/weapon/storage/bag/food = 5,
```
Comment (jknpj) [inline code/game/machinery/vending.dm]: There's no need for this ',' since this is the list's end

**Review by jknpj — APPROVED** [review]

**Comment by kilozombie** [comment]
Fuck yek
This also makes poutine oceans and poutine citadels actually usable and presentable without needing to bake a cake and cut it up and eat the slices for plates

**Comment by Pathid** [comment]
I like this, but at the same it's too bad there's no "clean" state for plates. They're going to come out of the vendor all crummy. That and they count as trash for the end score.

**Comment by TurboGay** [comment]
I was thinking about maybe taking the crumbs off the sprite and adding a clean plate item down the line @Pathid since that bothered me a bit too.

**Pull request merged**